Tests/MongoSwiftTests/AsyncAwaitTests.swift

```diff
@@ -224,6 +224,11 @@ final class MongoCursorAsyncAwaitTests: MongoSwiftTestCase {
 
     // Test that a tailable cursor that is continually polling the server can be killed by cancelling the parent Task.
     func testTailableCursorHandlesTaskCancellation() throws {
+        guard !MongoSwiftTestCase.serverless else {
```

Comment:
I bumped the Swift version used for serverless tests from 5.4 to 5.6, so this test started running against serverless, and failing due to the capped collection usage.